addons/mail/static/tests/discuss/core/web/discuss.test.js

```diff
@@ -167,6 +167,22 @@ test("Chat is pinned on other tabs when joined", async () => {
     await contains(`${env2.selector} .o-mail-DiscussSidebar-item`, { text: "Jerry Golay" });
 });
 
+test("Auto-open OdooBot chat when opening discuss for the first time", async () => {
+    // Odoobot chat has onboarding for using Discuss app.
+    // We assume pinned odoobot chat without any message seen means user just started using Discuss app.
+    const pyEnv = await startServer();
```

Comment:
I added a small comment to clarify the heuristic being used, as this isn't obvious from how the test is setup this leads to desired behavior.